common/strings/split.cc

```diff
@@ -32,4 +32,27 @@ std::vector<absl::string_view> SplitLines(absl::string_view text) {
   return lines;
 }
 
+// An zero-length absl::StrSplit delimiter that will match just after a
```

Comment:
It is used in a few places where it is useful to have unified lines (i.e. all with `\n` or all without `\n`) in order to avoid additional checks just for the last line.